tests/test_main.py

```diff
@@ -84,9 +84,9 @@ def test_create_person_list_has_wife_and_wife_have_husband(
 
 
 def test_create_person_list_has_no_wife(people_data, created_person_list):
-    assert hasattr(created_person_list[1], "wife") is False, (
+    assert created_person_list[1].wife is None, (
         f"Person with 'name' {created_person_list[1].name} should not have "
-        f"attribute wife"
+        f"attribute 'wife' or it should be None"
```

Comment:
The assertion message here is slightly misleading. It suggests that the 'wife' attribute should not exist or be None, but the code only checks if it is None. Ensure that the test accurately reflects the intended behavior.